emsdk.py

```diff
@@ -1038,8 +1033,7 @@ def quote_parens(x):
 def xcode_sdk_version():
   try:
     output = subprocess.check_output(['xcrun', '--show-sdk-version'])
-    if sys.version_info >= (3,):
-      output = output.decode('utf8')
+    output = output.decode('utf8')
     return output.strip().split('.')
```

Comment:
Possible alternative:
```suggestion
    return output.decode('utf8').strip().split('.')
```